docs/events.md

```diff
@@ -66,13 +72,19 @@ Components have the ability to bind to special inserted and removed events that
 }
 ```
 
+You can still bind to these special events by using the
+[can-3-4-compat](https://www.npmjs.com/package/can-3-4-compat) package, but this
+is deprecated in favor of the new [can-component/connectedCallback] API. See
+more information about migrating away from the inserted and removed events in
+the [migrate-4] guide.
+
```

Comment:
Here’s what this section looks like now:
<img width="788" alt="screen shot 2018-06-21 at 10 05 51 pm" src="https://user-images.githubusercontent.com/10070176/41758711-59e15660-759f-11e8-9a36-a13374761710.png">